app/src/main/res/layout/timetable_lesson.xml

```diff
@@ -75,15 +75,16 @@
                     android:layout_height="12dp"
                     android:layout_gravity="center_vertical"
                     android:layout_marginHorizontal="8dp"
-                    android:visibility="@{unread ? View.VISIBLE : View.GONE}"
-                    android:background="@drawable/unread_red_circle" />
+                    android:layout_marginTop="4dp"
```

Comment:
Why are you removing this margin?

Reply:
...now it looks like you're adding this margin..? Or maybe GitHub is broken, but to me it looks like a new line.